server/.husky/pre-commit

```diff
@@ -0,0 +1,6 @@
+#!/bin/sh
+. "$(dirname "$0")/_/husky.sh"
+
+npm test
+npx lint-staged
+npx pretty-quick --staged
```

Comment:
There are some changes to do, because of this: https://typicode.github.io/husky/#/?id=custom-directory, we have to do some "hack":
1. Here we have to add `cd server` on line 3, to let know that we have to do this in server folder. 

Beside that I think it is better to use our custom command of eslint with our rules (I didnt find that lint-staged is considering `.eslintrc` file). This is `npm run lint:eslint` command. Pretty-quick is good thing but I think it would be better to install it separately, then add script in `package.json`, for example: `"pretty-quick": "pretty-quick --staged"` and run this script here instead of installing it on every commit.

Another thing: Have in mind that for now script npm test always return error `"exit 1"`. so there are two options to consider: first, delete npm test from husky while there is no tests or change npm test command.